app/helpers/admin/products_helper.rb

```diff
@@ -14,6 +14,7 @@ def prepare_new_variant(product, producer_options)
       # e.g producer_options = [['producer name', id]]
       product.variants.build do |new_variant|
         new_variant.supplier_id = producer_options.first.second if producer_options.one?
+        new_variant.tax_category_id = product.variants.first.tax_category_id
```

Comment:
worth adding a check to see if variant exists? Ive noted in UI that a minimum of one variant always exists

Reply:
The UI prevents you from deleting a variant if there is only one left, and currently when creating a new product a variant will be created by default. So I think it's fair to assume there will always be one variant.